data/transition-sites/mod_sabre.yml

```diff
@@ -0,0 +1,6 @@
+---
+site: mod_sabre
+whitehall_slug: ministry-of-defence
+homepage: https://www.gov.uk/government/organisations/ministry-of-defence
```

Comment:
The `homepage` is used as the redirect location for the homepage of the old site - do we want to redirect users to somewhere more specific than the organisation homepage?